docs/content/posts/2023-11-23-release-kubernetes-operator-1.7.0.md

```diff
@@ -0,0 +1,81 @@
+---
+title:  "Apache Flink Kubernetes Operator 1.7.0 Release Announcement"
+date: "2023-11-23T08:00:00.000Z"
+authors:
+- gyfora:
+  name: "Gyula Fora"
+  twitter: "GyulaFora"
+aliases:
+- /news/2023/11/23/release-kubernetes-operator-1.7.0.html
+---
+
+The Apache Flink community is excited to announce the release of Flink Kubernetes Operator 1.7.0! The release introduces a large number of improvements to the autoscaler, including a complete decoupling from Kubernetes to support more Flink environments in the future. It's important to call out that the release explicitly drops support for Flink 1.13 and 1.14 as agreed by the community.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your feedback with the community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or [JIRA](https://issues.apache.org/jira/browse/flink)! We hope you like the new release and we’d be eager to learn about your experience with it.
+
+## Flink Version Support Policy Change
+
+Previously the operator only added more and more supported Flink versions without a policy to remove support for these in the future. This resulted in a lot of legacy codepaths already in the core logic.
+
+To keep things clean and easy to move forward at a reasonable pace, the community decided to adopt a new version support policy.
+
+{{< hint danger >}}
+Starting from 1.7.0 the operator will only support the last 4 Flink minor versions corresponding to the date of the operator release.
+For 1.7.0 this translates to: 1.18, 1.17, 1.16, 1.15
+{{< /hint >}}
+
+The operator will simply ignore changes made to resources with unsupported Flink versions. This also means that resources with unsupported versions are not possible to delete once the operator is upgraded. To temporarily work around this, users can upgrade the Flink version of the resource before trying to delete it.
+
+## Highlights
+
+### Decoupled Autoscaler Module
+
+[TODO]
```

Comment:
```suggestion
Starting from 1.7.0, Autoscaler is decoupled from kubernetes-operator. `flink-autoscaler` is an 
independent module of `flink-kubernetes-operator`, but it does not contain any `kubernetes-related` 
dependencies. `flink-autoscaler` defines a series of generic interfaces, read 
[Extensibility of Autoscaler](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-1.7/docs/custom-resource/autoscaler/#extensibility-of-autoscaler)
get more information.

In addition, the first version of `Autoscaler Standalone` is already supported. It supports single 
Flink cluster, it can be any type of Flink cluster, includes: `Flink Standalone Cluster`, `MiniCluster`, 
`Flink yarn session cluster`, `Flink yarn application cluster`, etc. `Autoscaler Standalone` runs as a
separate Java Process and is very simple, read [Autoscaler Standalone](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-1.7/docs/custom-resource/autoscaler/#autoscaler-standalone)
to see how to run it.
```

How about it?